components/layout/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+
+import { Stack, Grid, Text } from '@chakra-ui/core'
+import styled from '@emotion/styled'
+
+const Root = styled(Grid)`
+  padding: 36px 24px;
+  color: #888;
+  background-color: #111;
```

Comment:
Please convert these (padding + colors) to chakra-ui style props.

https://chakra-ui.com/style-props